packages/project-disaster-trail/src/components/Game/TaskScreen/SolveScreen.js

```diff
@@ -107,10 +118,12 @@ class SolveScreen extends PureComponent {
           step="Choose a task"
           durationLength={taskDuration / 1000}
         />
-        <OrbManager
-          possibleItems={playerKitItems}
-          onOrbSelection={this.onItemSelection}
-        />
+        <GUIStyle>
+          <OrbManager
+            possibleItems={possibleItems}
+            onOrbSelection={this.onKitItemSelection}
+          />
+        </GUIStyle>
```

Comment:
I feel like we maybe should rename the GUIStyle component? Or is this naming convention still helpful?

Reply:
GUI styles is not all that meaningful, I'll rename it.